Source/UnityVersionControl/Private/UnityVersionControlOperations.h

```diff
@@ -0,0 +1,582 @@
+// Copyright (c) 2023 Unity Technologies
+
+// Specialization of classes defined in Engine\Source\Developer\SourceControl\Public\SourceControlOperations.h
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "IUnityVersionControlWorker.h"
+#include "UnityVersionControlRevision.h"
+#include "UnityVersionControlState.h"
+#include "ISourceControlOperation.h"
+#include "SourceControlOperations.h"
+
+#include "Runtime/Launch/Resources/Version.h"
+
+#if ENGINE_MAJOR_VERSION == 5
+#include "UnityVersionControlChangelist.h"
+#include "UnityVersionControlChangelistState.h"
+#endif
+
+class FUnityVersionControlProvider;
+
+/**
+ * Internal operation used to revert checked-out unchanged files
+*/
+// NOTE: added to Engine in Unreal Engine 5 for changelists
+class FPlasticRevertUnchanged final : public ISourceControlOperation
```

Comment:
Question: Should we change references to `Plastic` in this file as well?

Reply:
That's a complex question: where to stop!
The plugin & module are public and needed to be renamed, so folder name & file names followed, but the rest is arbitrary.

I already renamed more than strictly needed for uniformity,
So it could make sense to also rename all the internal "class" and "struct" that starts with "FPlastic", though should I rename "FPlastic" to "FUnityVCS"?
I ended up with too many questions and kept only the easy "module + file name & associate classes"

Reply:
Agreed, you already renamed most of the references, so it's not a big deal at all